src/App.jsx

```diff
@@ -0,0 +1,83 @@
+import { DetailsPanel } from "./forecast&highlights/DetailsPanel";
+import { CurrentWeatherPanel } from "./search&currentWeather/CurrentWeatherPanel";
+import { createRoot } from "react-dom/client";
+import { useEffect, useState } from "react";
+
+const App = () => {
+  const [isActive, setIsActive] = useState(false);
+  const [location, setLocation] = useState("");
+  const [units, setUnits] = useState("metric");
+  const [lat, setLat] = useState("50.45");
```

Comment:
Combine this state or lat & lon into 1 unit, because every time you update lat you also update lon